xds/internal/balancer/cdsbalancer/cdsbalancer.go

```diff
@@ -435,13 +435,13 @@ func (b *cdsBalancer) run() {
 				// SubConn updates are passthrough and are simply handed over to
 				// the underlying cluster_resolver balancer.
 				if b.childLB == nil {
-					b.logger.Errorf("xds: received scUpdate {%+v} with no cluster_resolver balancer", update)
+					b.logger.Errorf("Received SubConn update with no child policy: %+v", update)
```

Comment:
When there is no child policy, I did not find it useful to specify the type of that non-existent child policy. Hence I got rid of the `cluster_resolver` name from the log. Also, if tomorrow, if we have a different child policy as the child of the `cds_experimental` LB policy, this comment would not need any updating.

Thinking a bit more on this, I feel that this error message (which is emitted by almost all of our LB policies in our tree) does not actually have a lot of value. This can happen around a small window when the child is removed, but a subConn created by the child has changed state (which in most cases would be moving towards `Shutdown`). There is no action taken for this event, and therefore I feel its value is minimal.